app/services/setting_registry.rb

```diff
@@ -159,7 +159,6 @@ def load_values(ignore_cache: false)
 
   def _add(name, category:, type:, default:, description:, full_name:, context:, value: nil, encrypted: false, collection: nil, options: {})
     select_collection_registry.add(name, collection: collection, **options) if collection
-    Foreman::Deprecation.deprecation_warning('3.3', "initial value of setting '#{name}' should be created in a migration") if value
```

Comment:
Here it means that the code should no longer handle a `value`. It should remove the method parameter `value` and anything that deals with it.

There is also an entry that masks this:
https://github.com/theforeman/foreman/blob/d4569bb4de9b6f2cc4972452e4e19b027c522bcf/config/as_deprecation_whitelist.yaml#L3
You should first make sure that's gone.